research-software-directory/RSD-as-a-service
# 303 mention types

# Improved mention type classification on the front end

Fixes #303 

Changes proposed in this pull request:
* When user search for mentions in datacite and crossref we classify the refrence to RSD types. 
* Specific mention types were not propperly classified by api. Mainly the classification from datacite was improved by including second type property.

How to test:
* `docker-compose down --volumes` to remove old
* `docker-compose build` to build solution
* `docker-compose up` to start
* login as any user
* create software 1
* in mentions section search for mentions by DOI and confirm the types extracted:
   * 10.3990/1.9789036550338 -> type: thesis,
   * 10.5194/egusphere-egu21-7797 -> type: other
   * 10.6084/m9.figshare.7053608.v1 -> type: presentation
   * 10.5281/zenodo.1045077 -> type: conference paper
   * 10.5281/zenodo.1043895 -> type: presentation
   * 10.5281/zenodo.1324069 -> type: report
   * 10.5281/zenodo.1064084 -> type: other

PR Checklist:
